R/mod_02_pre_process.R

```diff
@@ -622,6 +629,48 @@ mod_02_pre_process_server <- function(id, load_data, tab) {
       }
     )
 
+    # Markdown report
+    output$report <- downloadHandler(
+      # For PDF output, change this to "report.pdf"
+      filename ="report.pdf",
+      content = function(file) {
+        # Copy the report file to a temporary directory before processing it, in
+        # case we don't have write permissions to the current working dir (which
+        # can happen when deployed).
+        #tempReport <- file.path(tempdir(), "test_workflow.Rmd")
+        #tempReport
+        #tempReport<-gsub("\\", "/",tempReport,fixed = TRUE)
+        markdown_location <-"C:/Users/bdere/Documents/GitHub/idepGolem/vignettes/test_workflow2.Rmd"
```

Comment:
This is hard coded to your address?

Reply:
@bderenge

Reply:
I've added a commit that both fixes the hard-coding location and also comments out the report downloading section. those areas will be added back later in separate pull request.